docs/src/filesystem.md

```diff
@@ -190,6 +191,24 @@ write (transiently, i.e. until the next reboot) to all top-level directories,
 including `/usr` and `/opt`, with symlinks to `/var` for content that should
 persist.
 
+This can be combined with `etc.transient` as well (below).
+
+## Enabling transient etc
+
+The default (per above) is to have `/etc` persist. If however you do
+not need to use it for any per-machine state, then enabling a transient
+`/etc` is a great way to reduce the amount of possible state drift. Set
+the
+
+```toml
+[etc]
+transient = true
+```
+
+option in `prepare-root.conf`.
```

Comment:
Specifying the full path of `/usr/lib/ostree/prepare-root.conf` may prevent readers from searching.  I did a CTRL+f as I forgot the path.